beacon-chain/sync/regular_sync.go

```diff
@@ -79,39 +79,39 @@ type RegularSync struct {
 
 // RegularSyncConfig allows the channel's buffer sizes to be changed.
 type RegularSyncConfig struct {
-	BlockAnnounceBufferSize      int
-	BlockBufferSize              int
-	BlockReqSlotBufferSize       int
-	BlockReqHashBufferSize       int
-	BatchedBufferSize            int
-	StateReqBufferSize           int
-	AttestationBufferSize        int
-	AttestationReqHashBufSize    int
-	UnseenAttestationsReqBufSize int
-	ExitBufferSize               int
-	ChainHeadReqBufferSize       int
-	CanonicalBufferSize          int
-	ChainService                 chainService
-	OperationService             operations.OperationFeeds
-	BeaconDB                     *db.BeaconDB
-	P2P                          p2pAPI
+	BlockAnnounceBufferSize     int
+	BlockBufferSize             int
+	BlockReqSlotBufferSize      int
+	BlockReqHashBufferSize      int
+	BatchedBufferSize           int
+	StateReqBufferSize          int
+	AttestationBufferSize       int
+	AttestationReqHashBufSize   int
+	AttestationsAnnounceBufSize int
+	ExitBufferSize              int
+	ChainHeadReqBufferSize      int
+	CanonicalBufferSize         int
+	ChainService                chainService
+	OperationService            operations.OperationFeeds
+	BeaconDB                    *db.BeaconDB
+	P2P                         p2pAPI
 }
 
 // DefaultRegularSyncConfig provides the default configuration for a sync service.
 func DefaultRegularSyncConfig() *RegularSyncConfig {
 	return &RegularSyncConfig{
-		BlockAnnounceBufferSize:      100,
-		BlockBufferSize:              100,
-		BlockReqSlotBufferSize:       100,
-		BlockReqHashBufferSize:       100,
-		BatchedBufferSize:            100,
-		StateReqBufferSize:           100,
-		ChainHeadReqBufferSize:       100,
-		AttestationBufferSize:        100,
-		AttestationReqHashBufSize:    100,
-		UnseenAttestationsReqBufSize: 100,
-		ExitBufferSize:               100,
-		CanonicalBufferSize:          100,
+		BlockAnnounceBufferSize:     100,
```

Comment:
Later on we can have a config var that's a default buffer size across the repo for channels

Reply:
agree